src/main.ts

```diff
@@ -19,6 +19,12 @@ async function main() {
         azPath = await io.which("az", true);
         await executeAzCliCommand("--version");
 
+        let cloud = core.getInput('cloud', { required: false });
+        if (!cloud){
+        }else{
+            await executeAzCliCommand(`cloud set --name "${cloud}"`);

```

Comment:
@aaronba please pass "true" as second argument to executeAzCliCommand. Refer other executeAzCliCommand calls

Reply:
Done.  See latest commit.
and please take a look at ServicePrincipalLogin.ts as well for PowerShell case.
(code was added after initial PR)